packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -15,6 +15,10 @@
 /* global chai, workbox */
 
 'use strict';
+const testServerGen = require('../../../../utils/test-server-generator.js');
```

Comment:
sorry, for the goof up, I managed to fix all the test cases. lets sync up on this sometime today